### ISSUE TITLE
Error when calling Popen because env's key/value is not a string.

### ISSUE THREAD

**Issue by tanin47** [google/clusterfuzz-tools#199]
```
Traceback (most recent call last):
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/.bootstrap/_pex/pex.py", line 328, in execute
    self._wrap_coverage(self._wrap_profiling, self._execute)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/.bootstrap/_pex/pex.py", line 260, in _wrap_coverage
    runner(*args)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/.bootstrap/_pex/pex.py", line 292, in _wrap_profiling
    runner(*args)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/.bootstrap/_pex/pex.py", line 371, in _execute
    return self.execute_entry(self._pex_info.entry_point)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/.bootstrap/_pex/pex.py", line 429, in execute_entry
    return runner(entry_point)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/.bootstrap/_pex/pex.py", line 447, in execute_pkg_resources
    return runner()
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/clusterfuzz/main.py", line 76, in execute
    command.execute(**arg_dict)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/clusterfuzz/stackdriver_logging.py", line 130, in wrapped
    func(*args, **kwargs)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/clusterfuzz/commands/reproduce.py", line 243, in execute
    reproducer.reproduce(iterations)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/clusterfuzz/reproducers.py", line 211, in reproduce
    _, output = self.reproduce_crash()
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/clusterfuzz/reproducers.py", line 411, in reproduce_crash
    self.run_gestures(process, display_name)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/clusterfuzz/reproducers.py", line 353, in run_gestures
    windows = self.find_windows_for_process(proc.pid, display_name)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/clusterfuzz/reproducers.py", line 330, in find_windows_for_process
    exit_on_error=False, print_output=False)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/clusterfuzz/common.py", line 295, in execute
    proc = start_execute(command, cwd, environment, print_output)
  File "/usr/local/google/home/tanin/projects/clusterfuzz-cmd-tool/.pants.d/python-setup/chroots/1d6e49134380eed35c7fd867eec9fc44da73daad/clusterfuzz/common.py", line 248, in start_execute
    preexec_fn=os.setsid)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1327, in _execute_child
    raise child_exception
TypeError: execve() arg 3 contains a non-string value
```

I think because some envs' keys and/or values are unicodes. See: http://stackoverflow.com/questions/13213676/subprocess-popen-execve-arg-3-contains-a-non-string-value

**Comment by tanin47** [comment]
```
import subprocess
subprocess.Popen('ls', env={1: 'test'})
```

The above fails with the same exception. It's not unicode. Some keys and values are not string.

**Comment by tanin47** [comment]
Fixed by #201
